From: Ben Hutchings <ben@decadent.org.uk>
Date: Wed, 29 Nov 2017 20:05:45 +0000
Subject: mm/page_alloc: Avoid ABI change in 4.9.65
Forwarded: not-needed

Commit d135e5750205 "mm/page_alloc.c: broken deferred calculation"
renamed pglist_data::static_init_size to static_init_pgcnt.  However
the field has the same semantics as were intended originally (and in
any case, the field is only used by built-in code).  Hide the renaming
from genksyms.

---
--- a/include/linux/mmzone.h
+++ b/include/linux/mmzone.h
@@ -673,7 +673,11 @@ typedef struct pglist_data {
 	 */
 	unsigned long first_deferred_pfn;
 	/* Number of non-deferred pages */
+#ifndef __GENKSYMS__
 	unsigned long static_init_pgcnt;
+#else
+	unsigned long static_init_size;
+#endif
 #endif /* CONFIG_DEFERRED_STRUCT_PAGE_INIT */
 
 #ifdef CONFIG_TRANSPARENT_HUGEPAGE
